src/plugins/analysis/software_components/internal/scan_signatures.py

```diff
@@ -0,0 +1,13 @@
+def get_scanned_software(yara_signature_file):
```

Comment:
function is to complex.
consider use of common_helper_files.get_string_list_from_file to avoid errors
and export parsing to another function (s.a.).